english/turing-test/ghostbusters/ghostbusters.md

```diff
@@ -2,4 +2,8 @@ The ghostbusters eventually blow up the marshmallow man and you fly off in a hug
 
 Marshmallow cushions your fall and you have safely escaped the danger. 
 
-Congrats for completing the adventure!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.